Sources/PackagePlugin/PluginInput.swift

```diff
@@ -149,6 +150,7 @@ fileprivate struct PluginInputDeserializer {
             target = ClangSourceModuleTarget(
                 id: String(id),
                 name: wireTarget.name,
+                kind: (kind == .executable ? .executable : (kind == .test ? .test : .regular)),
```

Comment:
nit: an extension to case one enum to the other could be nicer to read / maintain

Reply:
Good point.  I did have a helper function with a switch at first, to keep the code close by, but it seemed clearer to do it inline.  But it's a good point if we add more cases.  I can make that change.

Reply:
Updated